src/libraries/Common/tests/System/Net/Http/PostScenarioTest.cs

```diff
@@ -80,11 +95,18 @@ public async Task PostEmptyContentUsingChunkedEncoding_Success(Configuration.Htt
         [Theory, MemberData(nameof(RemoteServersMemberData))]
         public async Task PostEmptyContentUsingConflictingSemantics_Success(Configuration.Http.RemoteServer remoteServer)
         {
+            if (remoteServer.HttpVersion.Major >= 2 && PlatformDetection.IsBrowser)
+            {
+                // [ActiveIssue("https://github.com/dotnet/runtime/issues/53018", TestPlatforms.Browser)]
+                return;
+            }
+
             await PostHelper(remoteServer, string.Empty, new StringContent(string.Empty),
                 useContentLengthUpload: true, useChunkedEncodingUpload: true);
         }
 
         [OuterLoop("Uses external servers")]
+        [SkipOnPlatform(TestPlatforms.Browser, "Not supported on Browser")]
```

Comment:
What isn't supported here? And same question for the other instances of `"Not supported on Browser"`.

Reply:
Actually, we don't know yet. all we get is `Failed to fetch` error, which I would like to address in https://github.com/dotnet/runtime/issues/53018
So I changed this to `ActiveIssue` rather.